app/marketing/models.py

```diff
@@ -44,7 +44,7 @@ def set_priv(self):
         from django.utils import timezone
 
         h = hashlib.new('ripemd160')
-        h.update("{}-{}-{}".format(h.hexdigest(), timezone.now(), self.email))
+        h.update(b"{}-{}-{}".format(h.hexdigest(), timezone.now(), self.email))
```

Comment:
@mbeacom this fixes the 500 error you were getting - I believe the python3 hashlib requires a bytestring now